src/runtime-rs/crates/resource/src/rootfs/mod.rs

```diff
@@ -51,16 +51,38 @@ impl RootFsResource {
         }
     }
 
+    #[allow(clippy::too_many_arguments)]
     pub async fn handler_rootfs(
         &self,
         share_fs: &Option<Arc<dyn ShareFs>>,
         hypervisor: &dyn Hypervisor,
         sid: &str,
         cid: &str,
+        root: oci::Root,
         bundle_path: &str,
         rootfs_mounts: &[Mount],
     ) -> Result<Arc<dyn Rootfs>> {
         match rootfs_mounts {
+            // if rootfs_mounts is empty
+            mounts_vec if mounts_vec.is_empty() => {
```

Comment:
This is because `mount_vec` will include the information of containerd snapshot directory, if we are using bundle to create container, we don't need the containerd snapshotter

Reply:
I am not sure whether `mounts_vec` is always empty when starting container with bundle. Maybe we should figure it out ...

Reply:
yes, I think so, because we don't rely on contained snapshotter if we start container in this way ?